app/src/main/res/layout/activity_event_register.xml

```diff
@@ -35,6 +36,36 @@
                 android:textStyle="bold" />
 
 
+            <TextView
+                android:id="@+id/eventRegisterStatusTV"
+                android:layout_width="match_parent"
+                android:layout_height="match_parent"
+                android:gravity="center"
+                android:paddingTop="5dp"
+                android:textSize="15dp"
+                android:textStyle="italic|bold" />
+
+            <LinearLayout
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_gravity="center_horizontal"
+                android:orientation="horizontal">
+
+                <TextView
+                    android:id="@+id/eventRegisterEventHost"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_marginTop="20dp"
+                    android:text="Event Hosted By:  " />
+
+                <TextView
+                    android:id="@+id/eventRegisterEventHostTV"
```

Comment:
Can we underline this or something, idk the best way to show this but its not obvious that clicking on the name goes to the charity details atm. 
Once Jacob fixes up the design we can adjust it, but just so everyone knows thats a thing it would be good to make the fact that this can direct to another page more apparent.

Reply:
Done in latest commit